--- asylum-0.2.3/Makefile~	2007-11-11 15:44:39.000000000 +0100
+++ asylum-0.2.3/Makefile	2007-11-11 15:43:55.000000000 +0100
@@ -1,14 +1,12 @@
 CC=gcc
-CFLAGS= -x c++ -O3 -funsigned-char \
-	-DRESOURCEPATH=\"$(INSTALLRESOURCEPATH)\" \
-	-DSCOREPATH=\"$(INSTALLHISCORES)\"
+CFLAGS= -x c++ -funsigned-char
 LIBS=-lm -lSDL -lSDL_mixer -lstdc++
 
 RESOURCES=data/Resources data/Ego data/Psyche data/Id data/Voices
 
-INSTALLBIN=/usr/games/asylum
-INSTALLRESOURCEPATH=/usr/share/games/asylum
-INSTALLHISCORES=/var/games/asylum
+INSTALLBIN=$(DESTDIR)/usr/bin/
+INSTALLRESOURCEPATH=$(DESTDIR)/usr/share/asylum
+INSTALLHISCORES=$(DESTDIR)/var/games/asylum
 
 INSTALLGROUP=games
 CHGRP=chgrp
@@ -27,21 +25,22 @@
 default: build
 
 $(INSTALLBIN): asylum Makefile
+	mkdir -p $(INSTALLBIN)
 	cp asylum $(INSTALLBIN)
-	$(CHGRP) $(INSTALLGROUP) $(INSTALLBIN)
-	$(CHMOD) g+s $(INSTALLBIN)
-	$(CHMOD) a+x $(INSTALLBIN)
+#	$(CHGRP) $(INSTALLGROUP) $(INSTALLBIN)
+#	$(CHMOD) g+s $(INSTALLBIN)
+#	$(CHMOD) a+x $(INSTALLBIN)
 
 install-resources: $(RESOURCES) Makefile
 	mkdir -p $(INSTALLRESOURCEPATH)
 	cp -r $(RESOURCES) $(INSTALLRESOURCEPATH)/
-	$(CHGRP) -R $(INSTALLGROUP) $(INSTALLRESOURCEPATH)/
-	$(CHMOD) -R a+rx $(INSTALLRESOURCEPATH)/
+#	$(CHGRP) -R $(INSTALLGROUP) $(INSTALLRESOURCEPATH)/
+#	$(CHMOD) -R a+rx $(INSTALLRESOURCEPATH)/
 
 install-hiscores: Makefile
 	mkdir -p $(INSTALLHISCORES)
-	$(CHGRP) $(INSTALLGROUP) $(INSTALLHISCORES)
-	$(CHMOD) g+rwx,o-rwx $(INSTALLHISCORES)
+#	$(CHGRP) $(INSTALLGROUP) $(INSTALLHISCORES)
+#	$(CHMOD) g+rwx,o-rwx $(INSTALLHISCORES)
 
 install-binary: $(INSTALLBIN)
 
@@ -53,7 +52,7 @@
 build: asylum
 
 asylum: asylum.c asylum.h Makefile
-	$(CC) $(CFLAGS) -o asylum asylum.c $(LIBS)
+	$(CC) $(OPTFLAGS) $(CFLAGS) -o asylum asylum.c $(LIBS)
 
 clean:
 	rm asylum
