--- asylum-0.2/Makefile~	2007-07-25 12:59:11.000000000 +0200
+++ asylum-0.2/Makefile	2007-07-25 12:59:09.000000000 +0200
@@ -6,9 +6,9 @@
 
 RESOURCES=data/Resources data/Ego data/Psyche data/Id data/Voices
 
-INSTALLBIN=/usr/games/asylum
-INSTALLRESOURCEPATH=/usr/share/games/asylum
-INSTALLHISCORES=/var/games/asylum
+INSTALLBIN=$(DESTDIR)/usr/bin/
+INSTALLRESOURCEPATH=$(DESTDIR)/usr/share/asylum
+INSTALLHISCORES=$(DESTDIR)/var/asylum
 
 INSTALLGROUP=games
 
@@ -16,21 +16,22 @@
 default: build
 
 $(INSTALLBIN): asylum Makefile
+	mkdir -p $(INSTALLBIN)
 	cp asylum $(INSTALLBIN)
-	chgrp $(INSTALLGROUP) $(INSTALLBIN)
-	chmod g+s $(INSTALLBIN)
-	chmod a+x $(INSTALLBIN)
+#	chgrp $(INSTALLGROUP) $(INSTALLBIN)
+#	chmod g+s $(INSTALLBIN)
+#	chmod a+x $(INSTALLBIN)
 
 $(INSTALLRESOURCEPATH): $(RESOURCES) Makefile
 	mkdir -p $(INSTALLRESOURCEPATH)
 	cp -r $(RESOURCES) $(INSTALLRESOURCEPATH)/
-	chgrp -R $(INSTALLGROUP) $(INSTALLRESOURCEPATH)/
-	chmod -R a+rx $(INSTALLRESOURCEPATH)/
+#	chgrp -R $(INSTALLGROUP) $(INSTALLRESOURCEPATH)/
+#	chmod -R a+rx $(INSTALLRESOURCEPATH)/
 
 $(INSTALLHISCORES): Makefile
 	mkdir -p $(INSTALLHISCORES)
-	chgrp $(INSTALLGROUP) $(INSTALLHISCORES)
-	chmod g+rwx,o-rwx $(INSTALLHISCORES)
+#	chgrp $(INSTALLGROUP) $(INSTALLHISCORES)
+#	chmod g+rwx,o-rwx $(INSTALLHISCORES)
 
 install-binary: $(INSTALLBIN)
 
