--- asylum-0.3.0/Makefile~	2009-02-23 14:15:13.000000000 +0100
+++ asylum-0.3.0/Makefile	2009-02-23 14:51:34.000000000 +0100
@@ -45,25 +45,21 @@
 	LIBS=-lm -lmingw32 -lSDL_mixer -lSDLmain -lSDL -mwindows
 endif
 ifeq ($(HOST),generic)
-	INSTALLBIN=/usr/games/asylum
-	INSTALLRESOURCEPATH=/usr/share/games/asylum
-	INSTALLHISCORES=/var/games/asylum
+	INSTALLBIN=$(DESTDIR)/usr/bin/
+	INSTALLRESOURCEPATH=$(DESTDIR)/usr/share/asylum
+	INSTALLHISCORES=$(DESTDIR)/var/games/asylum
 endif
 
 default: build
 
 ifneq ($(HOST),mingw)
 $(INSTALLBIN): asylum$(EXE) Makefile
+	mkdir -p $(INSTALLBIN)
 	cp asylum$(EXE) $(INSTALLBIN)
-	$(CHGRP) $(INSTALLGROUP) $(INSTALLBIN)
-	$(CHMOD) g+s $(INSTALLBIN)
-	$(CHMOD) a+x $(INSTALLBIN)
 
 install-resources: $(RESOURCES) Makefile
 	mkdir -p $(INSTALLRESOURCEPATH)
 	cp -r $(RESOURCES) $(INSTALLRESOURCEPATH)/
-	$(CHGRP) -R $(INSTALLGROUP) $(INSTALLRESOURCEPATH)/
-	$(CHMOD) -R a+rX $(INSTALLRESOURCEPATH)/
 
 install-hiscores: Makefile
 	mkdir -p $(INSTALLHISCORES)
@@ -71,8 +67,6 @@
 	touch $(INSTALLHISCORES)/PsycheHighScores
 	touch $(INSTALLHISCORES)/IdHighScores
 	touch $(INSTALLHISCORES)/ExtendedHighScores
-	$(CHGRP) -R $(INSTALLGROUP) $(INSTALLHISCORES)/*
-	$(CHMOD) -R 660 $(INSTALLHISCORES)/*
 
 install-binary: $(INSTALLBIN)
 
